src/screens/OrganizationEvents/CustomRecurrenceModal.tsx

```
@@ -0,0 +1,596 @@
+        show={customRecurrenceModalIsOpen}
+        onHide={hideCustomRecurrenceModal}
+        centered
+      >
+        <Modal.Header>
+          <p className={styles.titlemodal}>{t('customRecurrence')}</p>
+          <Button
+            variant="danger"
+            onClick={hideCustomRecurrenceModal}
+            data-testid="customRecurrenceModalCloseBtn"
+          >
+            <i className="fa fa-times"></i>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)